Orso.Arpa.Persistence/Seed/SelectValueSeedData.cs

```diff
@@ -359,29 +359,29 @@ public static IList<SelectValue> SelectValues
         /// </summary>
         public static SelectValue Contest => new(Guid.Parse("95de5380-4027-4b73-b4db-3697aba5ba38"), "Contest", string.Empty);
 
-        public static SelectValue Amateur => new(Guid.Parse("3f93768e-ac24-4741-9eb8-49d1e8e4a6e1"), "Amateur", string.Empty);
+        public static SelectValue Amateur => new(Guid.Parse("f036bca9-95d4-4526-b845-fff9208ab103"), "Amateur", string.Empty);
```

Comment:
warum hast du hier überall neue Ids vergeben? Das führt zu einer Neuanlage der Seeddaten und wir haben die dann doppelt in der db

Reply:
reverted